src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -51,8 +51,12 @@
     private static final Map<String, List<Integer>> CHECK_TO_ANCESTOR_NODES_MAP = new HashMap<>();
 
     static {
+        CHECK_TO_ANCESTOR_NODES_MAP.put("ArrayTrailingComma",
+                Arrays.asList(TokenTypes.ARRAY_INIT));
         CHECK_TO_ANCESTOR_NODES_MAP.put("AvoidNestedBlocks",
                 Arrays.asList(TokenTypes.SLIST));
+        CHECK_TO_ANCESTOR_NODES_MAP.put("CommentsIndentation",
+                Arrays.asList(TokenTypes.METHOD_DEF));
```

Comment:
Comments can be anywhere - on a variable, if-statement, try-statement, class, enum, etc. 
METHOD_DEF doesn't cover all these cases, and sometimes it is bigger scope than needed. 
For example, this will trigger violations for both comments, but should be only "comment 1":
```diff
void m() {
  if (asd) {
+// comment 1
  }

  if (asd) {
// comment 2
  }
}
```

Reply:
@baratali is ok to use SLIST, then `if`, `try`, `method` all can be covered. then add `class` and `enum`

Reply:
ok, let's try it. Also please add `INTERFACE_DEF` and `RECORD_DEF`

Reply:
ok, done

Reply:
>  Add more tests for edge cases.

This is not done. You have so many different nodes, and only one test for a variable. Also please add example from my previous comment https://github.com/checkstyle/patch-filters/pull/321#discussion_r478568767

Reply:
@baratali add tests

Reply:
> Also please add example from my previous comment #321 (comment)

@HuGanghui , why didn't you add my example to the test?

Reply:
@baratali sorry, add it.